### PR TITLE
Update to use PHPUnit 10 syntax

### PR DESCRIPTION
**Pull request by samsonasik**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Updated to latest PHPUnit 10 syntax version based on defined in composer.json

### CODE REVIEW

**Comment by samsonasik** [comment]
Ready for review 👍

File: psalm-baseline.xml [hunk 1]
```diff
@@ -117,18 +117,43 @@
...
+    </PossiblyUnusedMethod>
+    <UndefinedAttributeClass>
+      <code><![CDATA[CoversClass]]></code>
+    </UndefinedAttributeClass>
   </file>
   <file src="test/Command/ConfigDumperCommandTest.php">
     <PossiblyUnusedMethod>
       <code><![CDATA[ignoreUnresolvedArguments]]></code>
     </PossiblyUnusedMethod>
+    <UndefinedAttributeClass>
```
Comment (gsteel) [inline psalm-baseline.xml]: Why the `UndefinedAttribute…` for `CoversClass` 🤔

Comment (samsonasik) [reply psalm-baseline.xml]: It seems missing use statement, I will add it when I back to pc

Comment (samsonasik) [reply psalm-baseline.xml]: I've added `PHPUnit\Framework\Attributes\CoversClass` to use statement and regenerate psalm baseline https://github.com/laminas/laminas-servicemanager/pull/248/commits/7a4e3643d77ecceedde3191d7e0b1aaba74d0db9 👍

**Review by gsteel — APPROVED** [review]
Thanks @samsonasik 👍

**Pull request merged**